### PR TITLE
Dont expose blocking calls externally, except for fluent api

### PR DESCRIPTION
**Pull request by slang25**

### CODE REVIEW

File: JustSaying.Tools/Commands/MoveCommand.cs [hunk 1]
```diff
@@ -62,9 +63,9 @@ public bool Execute()
             return true;
         }
 
-        private void EnsureQueueExists(SqsQueueByName queue)
+        private async Task EnsureQueueExists(SqsQueueByName queue)
```
Comment (JosephWoodward) [inline JustSaying.Tools/Commands/MoveCommand.cs]: Should this not be called `EnsureQueueExistsAsync`?

Comment (slang25) [reply JustSaying.Tools/Commands/MoveCommand.cs]: Thanks @JosephWoodward, this has been addressed

**Comment by JosephWoodward** [comment]
@slang25 Done 👍

**Comment by slang25** [comment]
@AnthonySteele You are right, at the moment they have been lifted up the fluent api methods, and we warn about this in the release notes (see first item under [known issues here](https://github.com/justeat/JustSaying/releases/tag/v5.0.0)).

This removes the other public apis with the blocking, that people may have accidentally called and get tripped up by. Pull async fluent api will come in a later release 👍

**Pull request merged**